paper/paper.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'DeepRank2: Mining 3D Protein Structures with Geometric Deep Learning'
+tags:
+  - Python
+  - PyTorch
+  - structural biology
+  - geometric deep learning
+  - 3D protein structures
+  - protein-protein interfaces
+  - single-residue variants
+authors:
+  - name: Giulia Crocioni
+    orcid: 0000-0002-0823-0121
+    corresponding: true
+    affiliation: 1
+    equal-contrib: true
+  - name: Dani L. Bodor
+    orcid: 0000-0003-2109-2349
+    affiliation: 1
+    equal-contrib: true
+  - name: Coos Baakman
+    orcid: 0000-0003-4317-1566
+    affiliation: 2
+    equal-contrib: true
+  - name: Daniel Rademaker
```

Comment:
Daniel T. Rademaker

Reply:
Farzaneh designed the class diagram. So I think she should have a higher place in the author list :)